README.md

```diff
@@ -185,57 +171,21 @@ QMAKE_LFLAGS += \
 Now, just run the build steps again, including the `qmake` command, as
 described above.
 
-#### Loading pugin
-
-In order to load the GDB-ImageWatch plugin, simply edit the `~/.gdbinit` file
-(create it if it doesn't exist) and append the following line:
-
-    source /path/to/gdb-imagewatch/gdb-imagewatch.py
-
-This way, GDB will automatically load the GDB imagewatch plugin every time it
-starts.
-
-### Ubuntu 16.04 Automated Installation without QtCreator
-
-The script `configure_ubuntu_16.sh` automates some of the installation steps
-for GDB ImageWatch on Ubuntu 16.04.  From the root directory of this project,
-    execute the following:
-
-    bash configure_ubuntu_16.sh
-
-Note that this script will not instal Qt 5.6+ and GDB with python 3 support. If
-you don't have these packages, please follow the instructions above.
-
-When you are done, follow the instructions below in the section `Testing your
-installation`.
-
-## Testing your installation
-
-After compiling the plugin, you can test it by opening a console in the
-installation folder and running the following command from the root project
-directory:
-
-    python3 /path/to/gdb-imagewatch/gdb-imagewatch.py --test
-
-If the installation was succesful, you should see the GDB ImageWatch window
-with the same `sample_buffer_1` and `sample_buffer_2` buffers from the image on
-the header of this page.
-
 ## Using plugin
 
-When GDB hits a breakpoint, the GDB ImageWatch window will be opened. You only
+When GDB hits a breakpoint, the Open Image Debugger window will be opened. You only
 need to type the name of the buffer to be watched in the "add symbols" input,
 and press `<enter>`.
 
-Alternatively, you can also invoke the GDB ImageWatch window directly from GDB
+Alternatively, you can also invoke the Open Image Debugger window directly from GDB
```

Comment:
How does it work for LLDB?

Reply:
This is a legacy instruction; now OID starts up automatically after the first debugger stop. It has been like this for a while now, I just never remembered to get rid of this instruction. Doing it now...